client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -46,50 +45,37 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
     >
       {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
         <form onSubmit={handleSubmit} className={classes.form} noValidate>
-          <TextField
+          <h1 style={{ textAlign: 'center' }}>Login</h1>
+          <CustomInput
             id="email"
-            label={<Typography className={classes.label}>E-mail address</Typography>}
-            fullWidth
-            margin="normal"
-            InputLabelProps={{
-              shrink: true,
-            }}
-            InputProps={{
-              classes: { input: classes.inputs },
-            }}
+            label="Email"
             name="email"
-            autoComplete="email"
-            autoFocus
-            helperText={touched.email ? errors.email : ''}
-            error={touched.email && Boolean(errors.email)}
+            type="text"
             value={values.email}
-            onChange={handleChange}
+            classes={classes}
+            errors={errors}
+            touched={touched}
+            placeholder="Your Email"
+            handleChange={handleChange}
           />
-          <TextField
+          <CustomInput
             id="password"
-            label={<Typography className={classes.label}>Password</Typography>}
-            fullWidth
-            margin="normal"
-            InputLabelProps={{
-              shrink: true,
-            }}
-            InputProps={{
-              classes: { input: classes.inputs },
-              endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
-            }}
+            label="Password"
+            name="password"
             type="password"
-            autoComplete="current-password"
-            helperText={touched.password ? errors.password : ''}
-            error={touched.password && Boolean(errors.password)}
             value={values.password}
-            onChange={handleChange}
+            classes={classes}
+            errors={errors}
+            touched={touched}
+            placeholder="Your password"
+            handleChange={handleChange}
           />
           <Box textAlign="center" marginTop={5}>
-            <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
+            <RequiredButton type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
```

Comment:
As mentioned in the file, properly better to just call this `AuthButton` or `CustomButton` if we anticipate to use it in other places. I can't think of using it in other places so maybe `AuthButton` is more suitable.